specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-10-01-preview/common-types.json

```diff
@@ -0,0 +1,71 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-01",
+    "title": "Cost Management common types"
+  },
+  "paths": {},
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Error response indicates that the service is not able to process the incoming request. The reason is provided in the error message. \n\nSome Error responses: \n\n * 429 TooManyRequests - Request is throttled. Retry after waiting for the time specified in the \"x-ms-ratelimit-microsoft.consumption-retry-after\" header. \n\n * 503 ServiceUnavailable - Service is temporarily unavailable. Retry after waiting for the time specified in the \"Retry-After\" header.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ProxyResource": {
```

Comment:
>ProxyResource [](http://example.com/codeflow?start=5&length=13)

Consider inheritng the common-types v2 ProxyResource definition

[https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/common-types/resource-management/v2/types.json)